anni/src/subcommands/repo.rs

```diff
@@ -238,6 +246,73 @@ fn repo_get_vgmdb(options: &RepoGetVGMdb, manager: &RepositoryManager, get: &Rep
     Ok(())
 }
 
+#[derive(Parser, Debug, Clone)]
+pub struct RepoGetCue {
+    #[clap(short = 'p', long)]
```

Comment:
Path should be a positional argument(?

Like this:
https://github.com/ProjectAnni/anni/blob/c92a9c23ea8754ebc804fcfdd80d6689292b2bbe/anni/src/subcommands/split.rs#L50

Reply:
I consider this corresponds to `vgmdb --catalog`

Reply:
`--catalog` in vgmdb also needs change, since --catalog is actually a required argument